skymap_scanner/config.py

```diff
@@ -53,14 +55,13 @@
 
 
 # physics strings
-INPUT_PULSES_NAME_MAP: Final[dict[str, str]] = {
-    "2021a": "SplitUncleanedInIcePulses",
-    "2023a": "SplitInIcePulses",
-    "2024a": "SplitInIcePulses",
+KeyNames = namedtuple('KeyNames', 'pulseseries l2_splinempe')
+INPUT_KEY_NAMES_MAP: Final[dict[str, KeyNames]] = {
+    "2021a": KeyNames("SplitUncleanedInIcePulses", "OnlineL2_SplineMPE"),
+    "2023a": KeyNames("SplitInIcePulses", "l2_online_SplineMPE"),
+    "2024a": KeyNames("SplitInIcePulses", "l2_online_SplineMPE"),
 }
-DEFAULT_INPUT_PULSES_NAME: Final = "SplitUncleanedInIcePulses"
-
-INPUT_PULSES_NAME = "SplitUncleanedInIcePulses"
+DEFAULT_INPUT_KEY_NAMES: Final = KeyNames("SplitUncleanedInIcePulses", "OnlineL2_SplineMPE")
```

Comment:
These are keyed by the `realtime_format_version`, right? Can you make it clear that they are, either in a comment or by renaming the variable(s)?

Reply:
Added a comment